sensor/common/filesystem/pipeline/pipeline.go

```diff
@@ -53,6 +51,10 @@ func (p *Pipeline) translate(fs *sensorAPI.FileActivity) *storage.FileAccess {
 		Timestamp: fs.GetTimestamp(),
 	}
 
+	if fs.GetProcess().GetContainerId() == "" {
+		access.Hostname = fs.GetHostname()
+	}
```

Comment:
[nit] isn't this redundant? I think `Hostname` is already set in the previous lines.